src/ARVRInterface.cpp

```diff
@@ -226,12 +158,15 @@ godot_transform godot_arvr_get_transform_for_eye(void *p_data, godot_int p_eye,
 	// :)
 	ret = *p_cam_transform;
 	ret = godot::api->godot_transform_operator_multiply(&ret, &reference_frame);
-	ret = godot::api->godot_transform_operator_multiply(&ret, &arvr_data->hmd_transform);
+	ret = godot::api->godot_transform_operator_multiply(&ret, &arvr_data->ovr->get_hmd_transform());
```

Comment:
This fails compiling for me with error:

```
src/ARVRInterface.cpp: In function 'godot_transform godot_arvr_get_transform_for_eye(void*, godot_int, godot_transform*)':
src/ARVRInterface.cpp:161:95: error: taking address of temporary [-fpermissive]
  ret = godot::api->godot_transform_operator_multiply(&ret, &arvr_data->ovr->get_hmd_transform());
```
I've solved it by assigning arvr_data->ovr->get_hmd_transform() to a variable first.

Reply:
Sounds like a similar issue like we had with returning vectors....

Reply:
I'm returning it as a const pointer to the variable, lets see if that works.

Reply:
Yep seems to work.